Program.cs

```diff
@@ -634,6 +634,9 @@ private static bool runCommand(string command, string args, bool useSolutionPath
                 WindowStyle = ProcessWindowStyle.Hidden
             };
 
+            if (version != null)
+                psi.EnvironmentVariables["VERSION"] = version;

```

Comment:
I mean this is a little ad-hoc... Sticking this in this generic helper is a little icky.

I'd probably do something like

```diff
diff --git a/Program.cs b/Program.cs
index 869914e..1956892 100644
--- a/Program.cs
+++ b/Program.cs
@@ -325,7 +325,11 @@ namespace osu.Desktop.Deploy
                     // osu.AppImage.zsync is AppImage update information file, that is generated by the tool
                     // more information there https://docs.appimage.org/packaging-guide/optional/updates.html?highlight=update#using-appimagetool
                     runCommand(appImageToolPath,
-                        $"\"{stagingTarget}\" -u \"gh-releases-zsync|ppy|osu|latest|osu.AppImage.zsync\" \"{Path.Combine(Environment.CurrentDirectory, "releases")}/osu.AppImage\" --sign", false, version);
+                        $"\"{stagingTarget}\" -u \"gh-releases-zsync|ppy|osu|latest|osu.AppImage.zsync\" \"{Path.Combine(Environment.CurrentDirectory, "releases")}/osu.AppImage\" --sign", false,
+                        new Dictionary<string, string>
+                        {
+                            ["VERSION"] = version
+                        });
 
                     // mark finally the osu! AppImage as executable -> Don't compress it.
                     runCommand("chmod", $"+x \"{Path.Combine(Environment.CurrentDirectory, "releases")}/osu.AppImage\"");
@@ -620,7 +624,7 @@ namespace osu.Desktop.Deploy
             solutionPath = path;
         }
 
-        private static bool runCommand(string command, string args, bool useSolutionPath = true, string? version = null)
+        private static bool runCommand(string command, string args, bool useSolutionPath = true, Dictionary<string, string>? environmentVariables = null)
         {
             write($"Running {command} {args}...");
 
@@ -634,8 +638,11 @@ namespace osu.Desktop.Deploy
                 WindowStyle = ProcessWindowStyle.Hidden
             };
 
-            if (version != null)
-                psi.EnvironmentVariables["VERSION"] = version;
+            if (environmentVariables != null)
+            {
+                foreach (var pair in environmentVariables)
+                    psi.EnvironmentVariables.Add(pair.Key, pair.Value);
+            }
 
             Process? p = Process.Start(psi);
             if (p == null) return false;

```

Reply:
I thought about doing something like that, but since it would be used only one time, I thought it would be better if it was simple. But I will implement it like that if you prefer it that way.

Reply:
I basically did the same thing as you. I hope it's fine, but maybe it'd be better if you made the commit since I did almost nothing.